java/src/jmri/Bundle.java

```diff
@@ -64,7 +64,7 @@
      * @return Internationalized text
      */
     static String getMessage(String key) {
-        return b.handleGetMessage(key);
+        return bundle.handleGetMessage(key);
```

Comment:
Please don't change the names of object in the Bundle classes unless you are going to change all 312 of them.

Reply:
Ouch! I assumed that NetBeans "Refactor" would fix it everywhere… (Obviously, I was wrong.)
Un-Refactored!